src/control/server/mgmt_pool.go

```diff
@@ -170,18 +201,51 @@ func (svc *mgmtSvc) PoolCreate(ctx context.Context, req *mgmtpb.PoolCreateReq) (
 
 		req.Ranks = system.RanksToUint32(reqRanks)
 	} else {
-		// Otherwise, create the pool across all available ranks in the system.
-		req.Ranks = system.RanksToUint32(allRanks)
+		// Otherwise, create the pool across the requested number of
```

Comment:
shouldn't "the requested number of" be "request number (or all if not)"?

Reply:
Updated comment.